src/index.js

```diff
@@ -0,0 +1,190 @@
+    select.value = 'default';
+  }
+};
+
+const resetCity = () => {
+  updateCityName(defaultCity);
+  cityInput.value = '';
+};
+
+const apiCalls = () => {
+  let lat, lon;
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: currentCityDisplay.textContent,
+      },
+    })
+    .then(response => {
+      lat = response.data[0].lat;
+      lon = response.data[0].lon;
+
+      findWeather(lat, lon);
+    })
+    .catch(error => {
+      console.log('could not retrieve location', error.data);
+    });
+};
+
+const findWeather = (latitude, longitude) => {
+  axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: {
+        lat: latitude,
+        lon: longitude,
+      },
+    })
+    .then(response => {
+      console.log(response.data);
+      const kelvTemp = response.data.main.temp;
+      state.temp = Number(((kelvTemp - 273.15) * (9 / 5) + 32).toFixed());
+      displayTemp();
+    })
+    .catch(error => {
+      console.log('error in API Call - weather'.error.data);
+    });
+};
+
+const inputCaseSensitive = city => {
+  const citySplit = city.split(' ').filter(el => el !== '');
+
+  const styleWord = word => {
+    return word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
+  };
+
+  if (citySplit.length === 1) {
+    return styleWord(citySplit[0]);
+  }
+
+  const styledWords = citySplit.map(styleWord);
+  return styledWords.join(' ');
+};
+
+const selectSky = skyValue => {
+  if (skyValue === 'sunny') {
+    currentSky.textContent = '☀️ 🌤 ☁️ ☀️ ☀️ ☀️ ☀️ ☁️ ☀️ 🌤 ☀️';
+  } else if (skyValue === 'cloudy') {
+    currentSky.textContent = '☁️ 🌤 ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️ 🌤 ☁️';
+  } else if (skyValue === 'rainy') {
+    currentSky.textContent = '🌧 🌦 🌈 ⛈ 🌧 💧 ⛈ 🌦 💧 🌈 🌧';
+  } else if (skyValue === 'snowy') {
+    currentSky.textContent = '🌨 ❄️ 🌨 ❄️ ❄️ 🌨 ❄️ ❄️ 🌨 ❄️';
+  } else currentSky.textContent = '';
+};
+
+const autoChangeSky = () => {
+  if (state.temp < 40) selectSky('snowy');
+  else if (state.temp < 50) selectSky('rainy');
+  else if (state.temp < 70) selectSky('cloudy');
+  else if (state.temp >= 70) selectSky('sunny');
+};
+
+const displayTemp = () => {
+  currentTemp.textContent = `${state.temp}`;
+  changeColor();
+  changeLandscape();
+  autoChangeSky();
+};
+
+displayTemp();
+
+const registerEventHandlers = () => {
+  upButton.addEventListener('click', setUpTemp);
+  downButton.addEventListener('click', setDownTemp);
+  resetCityButton.addEventListener('click', () => {
+    resetCity(), apiCalls();
+  });
+  realTimeButton.addEventListener('click', () => {
+    updateCityName(cityInput.value);
+    apiCalls();
+  });
+  cityInput.addEventListener('keypress', event => {
+    if (event.key === 'Enter') {
+      realTimeButton.click();
+    }
+  });
+  skySelector.addEventListener('change', () => selectSky(skySelector.value));
+
+  window.onload = event => {
+    apiCalls();
+    resetCity();
+  };
+};
+
+document.addEventListener('DOMContentLoaded', registerEventHandlers);
```

Comment:
Beautiful work adding all of your event listeners to a well-organized `registerEventHandlers` function!